### PR TITLE
Build cacerts during build from Mozilla's list, take 2

### PR DESCRIPTION
**Pull request by aahlenst**
Generates the CA certificate bundle at build time from Mozilla's list of trusted CA certificates.

For context see https://github.com/AdoptOpenJDK/openjdk-support/issues/13. TSC approval is recorded in https://github.com/AdoptOpenJDK/openjdk-support/issues/13#issuecomment-635400251.

There was a previous attempt that failed (#2033, #2083) because I forgot to lock the keystore format to JKS/SUN 🤦. Adding tests to ensure that everything works is being tracked in https://github.com/AdoptOpenJDK/openjdk-tests/issues/1963. Ideally, we wait for the tests to appear before merging this one.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -186,6 +186,9 @@ if using docker, keep the container after the build.
 --make-exploded-image
 creates an exploded image (useful for codesigning jmods). Use --assemble-exploded-image once you have signed the jmods to complete the packaging steps.
 
+--custom-cacerts <true|false>
```
Comment (karianna) [inline README.md:189]: Should this be really called use-mozilla-certs?

File: makejdk-any-platform.1 [hunk 2]
```diff
@@ -166,8 +166,7 @@ For reference, OpenJDK version numbers look like 1.8.0_162-b12 (for Java 8) or
 This is typically used in conjunction with \fB<-b>\fR.
 .TP
 .BR \-\-use-jep319-certs
-Use certs defined in JEP319 in Java 8/9. This will increase the volume of traffic
-downloaded, however will provide an uptodate ca cert list.
+Use certs defined in JEP319 in Java 8/9. Deprecated, has no effect.
```
Comment (karianna) [inline makejdk-any-platform.1:169]: Also need the custom certs flag added to this file

File: sbin/prepareWorkspace.sh [hunk 3]
```diff
@@ -505,44 +505,15 @@ checkingAndDownloadingFreeType() {
   fi
 }
 
-# Download our security certificates
-downloadCerts() {
-  local caLink="$1"
+# Generates cacerts file
+prepareCacerts() {
```
Comment (karianna) [inline sbin/prepareWorkspace.sh:509]: prepareMozillaCacerts?

**Review by karianna — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (aahlenst) [reply README.md:189]: Doesn't really matter to me. Before I make the change: What if we ever decide to include another certificate?

**Comment by sxa** [comment]
@aahlenst What's the situation with this - are you looking to move it forward again now that the October releases have shipped? Are there any blockers?

**Comment by aahlenst** [comment]
As I said, we need the tests first. Tests are there, but not integrated. Waiting for Shelley and myself to work this out. Do not expect any movement from my side before late next week.

**Comment by karianna** [comment]
@aahlenst small merge conflict now as well

**Comment by aahlenst** [comment]
Now that https://github.com/AdoptOpenJDK/openjdk-tests/pull/2049 has landed, this is safe to merge as soon as the releases are done and the tests were successful. https://github.com/AdoptOpenJDK/openjdk-tests/issues/1963#issuecomment-725061131 demonstrates that the test detect a broken trust store.

File: makejdk-any-platform.1 (hunk 2 above)
Comment (aahlenst) [reply makejdk-any-platform.1:169]: Done

**Review by karianna — APPROVED** [review]
LGTM

**Comment by M-Davies** [comment]
run tests

**Comment by M-Davies** [comment]
> # 🟠 PR TESTER RESULT 🟠
> ❎ Some pipelines failed or the job was aborted! ❎
> _See the `pipeline-build-check` below for more information..._

One annoying network issue on a mac machine (unrelated to this PR)

**Review by M-Davies — APPROVED** [review]
Since the tests passed, I'm happy to merge this 👍

**Pull request merged**